### DISCUSSION TITLE
Resource URI in azure monitor

### DISCUSSION THREAD

**Discussion by Savitha-Amaldev** [kedacore/keda#2018] [Q&A / Need Help] [answered]
I'm trying to deploy a scaled object using the azure monitor trigger and keda . I have seen the sample scaled object defined on the keda website : https://keda.sh/docs/2.4/scalers/azure-monitor/  But I don't know how to get the URI of the azure monitor. I'm very new to these and I'm totally stuck on this. Can someone please help me in figuring out what needs to be done. Basically ,I don't have clarity on which resource's URI is asked here and how to get that?

**Comment by tomkerkhove** [comment, marked as answer]
This is the URI of the resource that you want to query, unfortunately the documentation on this is fairly limited.

It relies on the following pattern:
`subscriptions/<sub>/resourceGroups/<rg>/providers/<azure-provider>/<provider-info>`

For example, if you want to target a Service Bus namespace, that would be:
`subscriptions/<sub>/resourceGroups/<rg>/providers/Microsoft.ServiceBus/namespaces/promitor-messaging`

What resource are you trying to scale on?

**Reply by Savitha-Amaldev** [reply]
I'm trying to scale a service on a particular namespace based on the count of requests, it receives

**Reply by tomkerkhove** [reply]
Service bus namespace or what namespace are you referring to? with requests do you mean Service Bus messages?

**Reply by Savitha-Amaldev** [reply]
Not service bus namespace. I have a custom namespace in my environment. I have a deployment on the same namespace. So once pods get created for this deployment, it actually processes the HTTP requests targetting my deployment(Similar to a web service). What I want to do is based on the request count, I want to increase or decrease the pod count. So the trigger that I chose was azure monitor as it's enabled on the current Kubernetes cluster where my deployment is created.  Can we achieve it in this way?

**Reply by tomkerkhove** [reply]
I'm not sure why you are looking at Azure Monitor then, to be honest.

We do not support HTTP based autoscaling out of the box, but you can use Prometheus metrics or have a look at our [add-on](https://github.com/kedacore/http-add-on).

**Reply by Savitha-Amaldev** [reply]
Makes sense!.. Thanks Tom
